pkgs/development/python-modules/jaxlib/default.nix

```diff
@@ -204,40 +213,60 @@ let
 
     removeRulesCC = false;
 
-    GCC_HOST_COMPILER_PREFIX = lib.optionalString cudaSupport "${cudatoolkit_cc_joined}/bin";
-    GCC_HOST_COMPILER_PATH = lib.optionalString cudaSupport "${cudatoolkit_cc_joined}/bin/gcc";
+    GCC_HOST_COMPILER_PREFIX = lib.optionalString cudaSupport "${backend_cc_joined}/bin";
+    GCC_HOST_COMPILER_PATH = lib.optionalString cudaSupport "${backend_cc_joined}/bin/gcc";
```

Comment:
Since touching anyway, consider maybe adding the `targetPrefix` (grep nixpkgs for examples). Ok to ignore bc out of scope

Reply:
Roger that, I'll take a stab at this once I get the build working

Reply:
i genuinely appreciate this pointer, this PR has already gotten quite large so i propose that we leave that for future work in the interest of keeping PRs small. Created https://github.com/NixOS/nixpkgs/issues/289427 to track